tests/unit/scheduler/graphql_mid/test_graphql_mutations.py

```diff
@@ -0,0 +1,28 @@
+import pytest
```

Comment:
We should have the header here as well as in:
`tests/unit/scheduler/graphql_mid/__init__.py`